PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -1536,25 +1536,25 @@ class PurchasesTests: XCTestCase {
             ]
         ])
 
-        var receivedPurchaserInfo: PurchaserInfo?
+        var receivedCustomerInfo: CustomerInfo?
 
         purchases?.restoreTransactions { (info, error) in
-            receivedPurchaserInfo = info
+            receivedCustomerInfo = info
         }
 
-        expect(receivedPurchaserInfo?.originalApplicationVersion).toEventually(equal("1.0"))
-        expect(receivedPurchaserInfo?.originalPurchaseDate).toEventually(equal(Date(timeIntervalSinceReferenceDate: 562288673)))
+        expect(receivedCustomerInfo?.originalApplicationVersion).toEventually(equal("1.0"))
+        expect(receivedCustomerInfo?.originalPurchaseDate).toEventually(equal(Date(timeIntervalSinceReferenceDate: 562288673)))
         expect(self.backend.userID).toEventuallyNot(beNil())
         expect(self.backend.postReceiptDataCalled).toEventuallyNot(beFalse())
     }
 
-    func testCachesPurchaserInfo() {
+    func testCachesCustomerInfo() {
         setupPurchases()
 
-        expect(self.deviceCache.cachedPurchaserInfo.count).toEventually(equal(1))
-        expect(self.deviceCache.cachedPurchaserInfo[self.purchases!.appUserID]).toEventuallyNot(beNil())
+        expect(self.deviceCache.cachedCustomerInfo.count).toEventually(equal(1))
+        expect(self.deviceCache.cachedCustomerInfo[self.purchases!.appUserID]).toEventuallyNot(beNil())
 
-        let purchaserInfo = self.deviceCache.cachedPurchaserInfo[self.purchases!.appUserID]
+        let purchaserInfo = self.deviceCache.cachedCustomerInfo[self.purchases!.appUserID]
```

Comment:
and here

Reply:
Just you wait until you re-review, I made so many naming changes in the tests after you pointed this out.